### PR TITLE
Convert single line if to if with braces

### PR DESCRIPTION
**Pull request by Alan-Jowett**
## Description

Parts of the code aren't conformant to the coding guidelines. Coding guidelines call for a single line statements in if/else/while/for statements to be wrapped in {}. 

## Testing

CI/CD

## Documentation

None.

### CODE REVIEW

File: tests/sample/bindmonitor.c [hunk 1]
```diff
@@ -67,24 +67,27 @@ find_or_create_process_entry(bind_md_t* ctx)
...
-    if (!entry)
+    if (!entry) {
         return entry;
+    }
 
 #if 0
     for (index = 0; index < 64; index++) {
-        if ((ctx->app_id_start + index) >= ctx->app_id_end)
-            break;
+        if ((ctx->app_id_start + index) >= ctx->app_id_end) {             break; }
```
Comment (dthaler) [inline tests/sample/bindmonitor.c:90]: move break to new line

File: tests/sample/bindmonitor_tailcall.c [hunk 2]
```diff
@@ -81,24 +81,27 @@ find_or_create_process_entry(bind_md_t* ctx)
...
-    if (!entry)
+    if (!entry) {
         return entry;
+    }
 
 #if 0
     for (index = 0; index < 64; index++) {
-        if ((ctx->app_id_start + index) >= ctx->app_id_end)
-            break;
+        if ((ctx->app_id_start + index) >= ctx->app_id_end) {             break; }
```
Comment (dthaler) [inline tests/sample/bindmonitor_tailcall.c:104]: move break to new line

**Review by dthaler — COMMENTED** [review]
Two whitespace nits, otherwise ok

File: tests/sample/bindmonitor.c (hunk 1 above)
Comment (Alan-Jowett) [reply tests/sample/bindmonitor.c:90]: Strange that clang formatting didn't catch this.

Comment (Alan-Jowett) [reply tests/sample/bindmonitor.c]: OK, this is commented out code, which doesn't get formatted automatically.

File: tests/sample/bindmonitor_tailcall.c (hunk 2 above)
Comment (Alan-Jowett) [reply tests/sample/bindmonitor_tailcall.c]: OK, this is commented out code, which doesn't get formatted automatically.

**Review by dv-msft — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Comment by gtrevi** [comment]
Would it be a good idea if we also add `InsertBraces` to `.clang-format` (does exactly this for conditions & loops)?

Was added here: https://github.com/llvm/llvm-project/commit/77e60bc42c48e16d646488d43210b1630cd4db49, though the [documentation](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) has some warnings to consider:

```
Warning

Setting this option to true could lead to incorrect code formatting due to clang-format’s lack of complete semantic information. 

As such, extra care should be taken to review code changes made by this option.
```

**Review by shankarseal — APPROVED** [review]

**Pull request merged**